staging/src/k8s.io/api/autoscaling/v2beta2/types.go

```diff
@@ -373,7 +373,8 @@ type HorizontalPodAutoscalerStatus struct {
 
 	// conditions is the set of conditions required for this autoscaler to scale its target,
 	// and indicates whether or not those conditions are met.
-	Conditions []HorizontalPodAutoscalerCondition `json:"conditions" protobuf:"bytes,6,rep,name=conditions"`
+	// +optional
+	Conditions []HorizontalPodAutoscalerCondition `json:"conditions,omitempty" protobuf:"bytes,6,rep,name=conditions"`
```

Comment:
Adding `omitempty` causes the serialization to skip the field if there are no conditions. Users of the HPA status might be expecting the field to be present, even if null. They could start crashing.
We also don't want to change how the server stores and serves the data.